src/emcpy/calcs/units.py

```diff
@@ -1,36 +1,102 @@
+__all__ = ['K_to_C', 'K_to_F', 'C_to_K', 'C_to_F', 'F_to_C', 'mps_to_MPH',
+           'mm_to_inches']
```

Comment:
do we not ever need `inches_to_mm`?

Reply:
yes there should probably be MPH to mps also.